src/test/java/ch/jalu/collectionbehavior/ListTest.java

```diff
@@ -240,103 +323,62 @@ void jdkCollectionsUnmodifiableList() {
         assertThat(list, instanceOf(RandomAccess.class)); // Because wrapped list is RandomAccess, too.
```

Comment:
* Sweep to check all old tests have been removed
 * Rename the remaining non-TestFactory tests to meaningfully describe what they test _additionally_